kitsune/wiki/facets.py

```diff
@@ -1,13 +1,15 @@
 import hashlib
+from datetime import datetime, timedelta
+from operator import itemgetter
 
 from django.conf import settings
 from django.core.cache import cache
 from django.db.models import Count
 
-from elasticsearch.exceptions import TransportError
-
 from kitsune.products.models import Topic
-from kitsune.wiki.models import Document, DocumentMappingType
+from kitsune.wiki.models import Document, HelpfulVote
+
+VOTES_CACHE_TIMEOUT = 24 * 60 * 60  # 24 hours
```

Comment:
It's a rather expensive query, and it doesn't need to be 100% up-to-date, since it just affects ordering - indeed in the previous implementation where the votes were stored in ES it wasn't. I'll add a comment explaining this.

Reply:
You can now use the new settings for cache here.